distribution.pm

```diff
@@ -141,8 +143,10 @@ sub script_run {
     }
     testapi::type_string "$cmd";
     if ($args{timeout} > 0) {
+        my $terminator = ((substr $cmd, -1) eq '&') ? '' : ';';
+        die "Terminator '&' found in script_run call. script_run can not check script success. Consider using 'type_string'." unless $terminator;
```

Comment:
At first I though we should make this a softfailure to have a smooth grace period. But this should anyways always result in a test failure, so it should be good. Only making this remark to point out, that we do not believe this has any destructive impact on existing test cases.

Reply:
Yes, I believe this has not any destructive impact on existing test cases. The definition of "softfailure" from http://open.qa/docs/#_jobs is "At least one known, non-critical issue has been found." but within os-autoinst-distri-opensuse we enforce softfailure's to link to existing issue tracker entries, e.g. bug reports in bugzilla, which we would not have here.